### ISSUE TITLE
Not all errors are emitted

### ISSUE THREAD

**Issue by DorianGrey** [TypeStrong/fork-ts-checker-webpack-plugin#9]
Hi there,

I'm currently investigating if it's possible to replace atl with ts-loader and this plugin in my projects, since the expectable (re-)build performance increase was / is quite impressive.
While testing, I've faced an issue with the emitted errors - they were incomplete.

The playground project I've tested the change on can be found [here](https://github.com/DorianGrey/react-ts-playground). The master branch still contains the atl version; I've set up an example for the issue on [this](https://github.com/DorianGrey/react-ts-playground/tree/ts-loader-test) branch.

In particular, I've changed [this line](https://github.com/DorianGrey/react-ts-playground/blob/ts-loader-test/src/app/i18n/i18n.ts#L1) so that it includes an unused import, which raise a compiler error w.r.t. the project's `tsconfig.json`.

**Expected behavior:**
Using `yarn build:dev` should emit an error like this:
```
ERROR in [at-loader] ./src/app/i18n/i18n.ts:1:24 
    TS6133: 'IntlShape' is declared but never used.
```
This is what happens using atl's `CheckerPlugin` or `transpileOnly: false` as option for ts-loader.

**Actual behavior:**
Build works fine, no error is emitted.

Did I miss something on the [configuration](https://github.com/DorianGrey/react-ts-playground/blob/ts-loader-test/webpack/_common.config.js#L60), or is this a bug?

**Comment by piotr-oles** [comment]
Hi!
Thank you for reporting this issue :) I've reproduced this case and found the problem.
To fix it, you have to change `tsconfig.json`:
```
30.      ]
31.    },
32. -  "files": [
33. -    "src/customTypings.d.ts"
34. -  ],
35.    "exclude": [
36.      "node_modules"
37.    ],
```
It's because this plugin uses `tsconfig.json` to get a list of files included in a build and your config includes only `src/customTypings.d.ts`. It's missing part of the documentation, I will add it soon. Unfortunately, it doesn't use webpack's module resolution because of performance. A full list of included files is available only on *emit* event so it's a little bit late.

**Comment by DorianGrey** [comment]
First of all - it works this way, and I really appreciate the results; it also seems to illustrate that I have to reconsider my current linter usage as well - I enabled your plugin's `tslint` by accident and it raised tons of issues that have not been raised before, although they are configured.

However, it's still a bit confusing that the `files` entry has to be removed completely - I added this file manually since it was also added this way in a different project with a similar setup (using atl), and that one failed to properly pick up this definition file otherwise. But if things work without having to add these entries manually, I'm totally fine with it :) 

I'll go back testing, then... 

Thanks!
